OrbitGl/TimeGraph.h

```diff
@@ -62,7 +62,12 @@ class TimeGraph {
   void SetMinMax(double a_MinTimeUs, double a_MaxTimeUs);
   void PanTime(int a_InitialX, int a_CurrentX, int a_Width,
                double a_InitialTime);
-  void HorizontallyMoveIntoView(const TextBox* text_box, double distance = 0.3);
+  enum VisibilityType {
```

Comment:
I'm not the expert here. It is ok using enum instead enum class here?

Reply:
Fixed.